pizza.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Pizza-order</title>
+    </head>
+    <body>
+        <header>
+            <h2>Pizza Order</h2>
+        </header>
+        <div><form action="https://www.google.ro/">
+            <label for="nume"><b>Your Name:</b></label>
+            <input type="text" name="nume" minlength="3" maxlength="10" placeholder="your name" required><br><br>
+            <label for="email"><b>Your email:</b></label>
+            <input type="email" name="email" placeholder="your email" required><br><br>
+            <label for="topping"><b>Topping:</b></label>
+            <input type="radio" name="topping" value="Supreme"><b>Supreme</b>
+            <input type="radio" name="topping" value="Vegetarian" checked ><b>Vegetarian</b>
+            <input type="radio" name="topping" value="Hawaian"><b>Hawaian</b><br><br>
+            <label for="sauce"><b>Sauce:</b></label>
+            <select name="sauce">
+                <option value="Tomato"><b>Tomato</b></option>
+                <option value="Sweet Chilli"><b>Sweet Chilli</b></option>
+                <option value="Tzatziki"><b>Tzatziki</b></option>
+            </select><br><br>
+            <label for="extras"><b>Extras:</b></label>
+            <input type="checkbox" name="extras" value="Cheese" checked><b>Cheese</b>
+            <input type="checkbox" name="extras" value="Onions" ><b>Onions</b>
+            <input type="checkbox" name="extras" value="Olives" ><b>Olives</b><br><br>
+            <label for="data de livrare"><b>Data de livrare</b></label>
+            <input type="date" name="data de livrare" min="2024-04-10"><br><br>
+            <label for="numar"><b>Numar:</b></label>
+            <input type="number" name="numar" min="1" max="7" value="1"><br><br>
+            <label for="delivery instructions"><b>Delivery Instructions:</b></label><br>
```

Comment:
este mai ok ca for sa nu contina spatii. De asemenea, el nu este legat de textarea prin id